openwisp_firmware_upgrader/admin.py

```diff
@@ -90,7 +90,7 @@ class CategoryFilter(MultitenantOrgFilter):
 
 
 @admin.register(load_model('Build'))
-class BuildAdmin(BaseVersionAdmin):
+class BuildAdmin(BaseAdmin):
```

Comment:
We don't want to remove reversion for Build model, but for `FirmwareBuild` model.

Reply:
**Correction:** After discussion with @nemesisdesign, we came to the conclusion that we want to remove `Build` model from reversion. Hence, the above change is correct. 

To fix tests, it is required to remove following snippet of code as well 

https://github.com/openwisp/openwisp-firmware-upgrader/blob/91fe0b20e6697d5236f3184da6e0fd38af75463b/openwisp_firmware_upgrader/admin.py#L53-L56